src/d3a/models/strategy/commercial_producer.py

```diff
@@ -29,7 +28,7 @@ def __init__(self, energy_rate=None):
         if energy_rate is not None and energy_rate < 0:
             raise ValueError("Energy rate should be positive.")
         super().__init__()
-        self.energy_per_slot_kWh = Q_(int(sys.maxsize), ureg.kWh)
+        self.energy_per_slot_kWh = int(sys.maxsize)
```

Comment:
Any particular reason to get rid of units?

Reply:
Yes, because it was the last place in the source code that they were used. I remember we decided some time ago to remove these, because they are useful only if you enforce these throughout the source code (as you can see here, the units are casted away by using .m when using them, while this should not be the case, the units should be used as they were).